src/ProjectTemplates/Microsoft.Extensions.AI.Templates/src/ChatWithCustomData/.template.config/template.json

```diff
@@ -52,6 +52,12 @@
       "type": "bind",
       "binding": "HostIdentifier"
     },
+    "assemblyName": {
+      "type": "derived",
+      "valueSource": "name",
+      "valueTransform": "identity", // Avoids transformation,
```

Comment:
[nit]
```suggestion
      "valueTransform": "identity", // Avoids transformation
```

Reply:
Thanks. Changed the approach to not need this; so the block is deleted.